charts/eks-anywhere-packages/values.yaml

```diff
@@ -85,4 +85,14 @@ controller:
       memory: 50Mi
     limits:
       cpu: 500m
-      memory: 300Mi
\ No newline at end of file
+      memory: 300Mi
+# Additional Variables to configure cronjob
+cronjob:
+  name: ecr-refresher
+  # -- ECR refresher repository name.
+  repository: "eks-anywhere-packages/ecr-token-refresher"
```

Comment:
seriously, this format might not be compatible with all registries where we have a repository named `eks-anywhere-packages` and another named `eks-anywhere-packages/ecr-token-refresher` logically it makes no sense

Reply:
Responded in an earlier comment. No strong preference on my end.

Reply:
going with ecr-token-refresher .